src/rules.c

```diff
@@ -59,6 +59,7 @@ void rule_apply(struct rule *r, struct notification *n)
                 n->markup = r->markup;
         if (r->icon_position != -1)
                 n->icon_position = r->icon_position;
+
```

Comment:
Try not to add random newlines.

Reply:
yeah, that was a leftover :slightly_smiling_face: I removed it.